deploy/manifests/sample-deployments/kong-enterprise-hybrid/cluster-tls/cluster.key

```diff
@@ -0,0 +1,6 @@
+-----BEGIN PRIVATE KEY-----
+MIG2AgEAMBAGByqGSM49AgEGBSuBBAAiBIGeMIGbAgEBBDAehvQUaVk959HUqh3V
+daHH+lOfRxulzDRsXl+jPJ858XHNgjHchyAlizQbP/2sS4ShZANiAAR1kV79trQi
+ZCMbOn2UHzBIQNHPGnAksZetKOf0MFuqxvvj3FqXUihD81mFA6tOJFDDriC1k7f6
+TqRNbd4sn+6bW7cCJ9SM7VgasC8GbugKXNFJpgz+UhE3hJUt/n3rXaA=
+-----END PRIVATE KEY-----
```

Comment:
Ditto comments on the other cert/key pair.

Reply:
Removed the certs and replaced with some "EXAMPLE" content